lesson2/task3.js

```diff
@@ -0,0 +1,11 @@
+var a = 6
+var b = 4
+
+if(a >= 0 && b >= 0){
+    alert(a - b);
+
+}else if(a < 0 && b < 0){
+    alert(a * b);
+}else if((a >= 0 && b < 0) || (a < 0 && b >= 0) ){
```

Comment:
Эта проверка здесь лишняя
```suggestion
} else {
```